duckduckgo/iOS
# Debug logs

Task/Issue URL: https://app.asana.com/0/856498667320406/1156940418495162
Tech Design URL: https://app.asana.com/0/856498667320406/1156661500134961
CC:

**Description**:
*Do not merge:* Debug build for in house investigation. 

**Steps to test this PR**:
1. Open the app.
2. Check logs.
3. Change date on the device -> New log file should be created.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
